SU2_CFD/include/solvers/CIncEulerSolver.hpp

```diff
@@ -38,7 +38,7 @@
  */
 class CIncEulerSolver : public CFVMFlowSolverBase<CIncEulerVariable, INCOMPRESSIBLE> {
 protected:
-  CFluidModel *FluidModel = nullptr;    /*!< \brief fluid model used in the solver */
+  vector<CFluidModel*> FluidModel;   /*!< \brief fluid model used in the solver. */
```

Comment:
Don't you want to put the fluid model in the base class? CEulerSolver has exactly the same variable.

Reply:
It is a bit different for the NEMO solver, it also has a "FluidModel" but of a different type.